helper-cli/src/main/kotlin/commands/StorageListCommand.kt

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (C) 2020 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package com.here.ort.helper.commands
+
+import com.beust.jcommander.DynamicParameter
+import com.beust.jcommander.JCommander
+import com.beust.jcommander.Parameter
+import com.beust.jcommander.Parameters
+
+import com.here.ort.helper.CommandWithHelp
+import com.here.ort.helper.common.IdentifierConverter
+import com.here.ort.model.Identifier
+import com.here.ort.model.config.OrtConfiguration
+import com.here.ort.model.config.ScannerConfiguration
+import com.here.ort.model.yamlMapper
+import com.here.ort.scanner.ScanResultsStorage
+import com.here.ort.utils.PARAMETER_ORDER_MANDATORY
+import com.here.ort.utils.PARAMETER_ORDER_OPTIONAL
+
+import java.io.File
+
+@Parameters(
+    commandNames = ["storage-list"],
```

Comment:
The command sounds as if it would list the storages themselves. How about something like "list-stored-results"?

Reply:
There will be at least one more command working with scan results, I would like them all to use the same prefix. What about e.g. "scan-results-list" and "scan-results-remove"? It would sound better to use "list-scan-results" and "remove-scan-results" but then the commands would not be listed next to each other.

Reply:
Or it could be a single command "scan-results" with separate parameters "--list" and "--remove"?

Reply:
I think I like your last suggestion with the single command that takes different parameters best best.

Reply:
I would like to do the renaming as part of my next PR tomorrow when I add the remove command.

Reply:
Ok.

Reply:
Note that all other commands are in `imperative` form, and thus I believe this one should be as well.

Reply:
Agreed, sub-command names should start with a verb. Maybe we can rename this to `list-storage` for now, and think about about a better name once the second sub-command that operates on scan results is introduced?

Reply:
I'd rather have a final name than to rename twice. Any suggestion for an imperative name that performs actions on scan results? Does it have to be imperative? `orth scan-results --list` sounds good to me. Otherwise we need to have separate commands like `orth list-scan-results` and `orth remove-scan-results` which would also be ok.

Reply:
I like your proposal of having separate commands with the names you suggested for lower complexity per command.

Reply:
While I would have had a *slight* preference for `orth scan-results --list` for being more future proof without exploding the number of sub-commands, I'm also fine with the separate commands.

Reply:
@sschuberth can you explain what you a referring to exactly by `more future proof ` ?

Reply:
In the sense that introducing more scan-result-related helper commands would not necessarily require to add yet more sub-command, but only new options to existing sub-commands.

Reply:
I'll go with `list-scan-results` then because I don't expect that many more commands than `remove-scan-results` will be added in future.

Reply:
Done. I went with `list-stored-scan-results` to better differentiate the command from the other list commands.